reverse-proxy-nginx.conf

```diff
@@ -53,6 +53,13 @@ http {
       proxy_set_header Upgrade $http_upgrade;
       proxy_set_header Connection $connection_upgrade;
     }
+
+    location /pgadmin/ {
```

Comment:
maybe we now want to expose ecamp only on the localhost interface?
but we can do that in a seprate pr